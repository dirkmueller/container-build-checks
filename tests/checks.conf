[General]
Vendor=org.opensuse
Registry=registry.opensuse.org
Registry+=registry.example.com

[Tags]
Allowed+=opensuse/*
Blocked+=naughty*
