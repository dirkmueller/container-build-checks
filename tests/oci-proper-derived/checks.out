Looking at oci-proper-derived.containerinfo
Release: 7.21
Tag: opensuse/example:latest
Tag: opensuse/example:5.7
Tag: opensuse/example:5.7.7.21
Tag: kubic/testtag:latest
Detected image specific label prefix: org.opensuse.example

container-build-checks done. Hints: 0 Warnings: 0 Errors: 0
Exited with 0
